Source/capture.cpp

```diff
@@ -81,65 +81,75 @@ bool __fastcall CapturePix(HANDLE hFile, WORD width, WORD height, WORD stride, B
     do {
         if (!height) {
             mem_free_dbg(pBuffer);
-            return 1;
+            return TRUE;
         }
         height--;
         BYTE *pBufferEnd = CaptureEnc(pixels, pBuffer, width);
         pixels += stride;
         writeSize = pBufferEnd - pBuffer;
     } while (WriteFile(hFile, pBuffer, writeSize, &lpNumBytes, 0) && lpNumBytes == writeSize);
-    return 0;
+
+    return FALSE;
 }
 
 BYTE *__fastcall CaptureEnc(BYTE *src, BYTE *dst, int width)
 {
     do {
-        BYTE rlePixel = *src++;
-        --width;
-
+        BYTE rlePixel = *src;
+        *src++;
         int rleLength = 1;
+
+        width--;
+
         while (rlePixel == *src) {
             if (rleLength >= 63)
                 break;
             if (!width)
                 break;
-            ++rleLength;
+            rleLength++;
 
-            --width;
-            ++src;
+            width--;
+            src++;
         }
 
-        if (rlePixel > 0xBF || rleLength > 1) {
-            *dst++ = rleLength | 0xC0;
+        if (rleLength > 1 || rlePixel > 191) {

```

Comment:
0xBF seem more clear, as we later or with 0xC0. Even better would be `rlePixel >= 0xC0`, but perhaps this is not bin exact?

Reply:
Idk, the or is only on rleLength and 191 is on rlePixel, but the hole thing is a bit strange with doing logical compares and bin operators on almost all varables and then feeding them to dst in the end.

Reply:
Oh, I didn't notice they were operating on different variables. Fair point then.